docker-compose.yml

```diff
@@ -156,6 +156,29 @@ services:
       - "traefik.frontend.rule=${TRAEFIK_FRONTEND_RULE:-}PathPrefix:/bety/"
       - "traefik.backend=bety"
 
+  # ----------------------------------------------------------------------
+  # Pecan-RStudio
+  # ----------------------------------------------------------------------
+  rstudio:
+    image: pecan/base:${PECAN_RSTUDIO_VERSION:-latest}
```

Comment:
I think it is best to use PECAN_VERSION since we want to have it go hand in hand with the running version of the stack.

Reply:
Make sense, good idea. Done